### ISSUE TITLE
Change States Parties table data to non-signatory conditional

### ISSUE THREAD

**Issue by budparr** [ABA-Center-for-Human-Rights/aba-icc#99]
See, Afghanistan

**Comment by budparr** [comment]
cc/ @kiphale

**Comment by budparr** [comment]
Okay, @kiphale here are the new set of conditionals for the States Parties table. Let me know if this looks right to you. It appears to be right as far as I can tell. (I just uploaded the changes so it may take a few minutes to show up on the site)


If there's a date entered for ```"Rome Statute Data | Date Signed"```

show: **Signature [date]**

---

If there's a date entered for ```"Rome Statute Data | Date Ratified or Acceded" ```

show: **Ratification or Accession [date]**
   
---


If there's a date entered for ```"Rome Statute Data | Date Signed"```
OR 
The ```"Rome Statute Data"``` Switch is turned on (green)
AND
If there's NOT a date entered for ```"Rome Statute Data | Date Ratified or Acceded"```

show: **Not Ratified or Acceded**

---
 
If there's NOT a date entered for ```"Rome Statute Data | Date Signed"```
AND
The ```"Rome Statute Data"``` Switch is NOT turned on (grey)

show: **Not a Signatory**

**Comment by kiphale** [comment]
@budparr To be honest hard to follow, so can I put into legal talk?

If there is a date for signature but no date for ratification/accession = Signatory
If there is a date for signature and date for ratification/accession = State Party
If there is no date for signature but a date for accession = State Party
If there is no date for signature and no date for accession = not a signatory (or non-State Party)

**Comment by kiphale** [comment]
@budparr 

And also, if not too problematic, if somebody did not sign but later joined, it should just say "date of accession". If someone signed and later joined, it should say "date of ratification".

**Comment by budparr** [comment]
You're making it too complicated. My conditionals have to cover two things: 

One, to show a date - so that's pretty literal (if there's a date there, show it). That covers the first two items.

and two, whether or not to show
```Not Ratified or Acceded```
or 
```Not a Signatory```

So, signatory and state party are assumed by the first two since there are dates.

**Comment by kiphale** [comment]
Ok...I was just putting into my thought pattern to see if you could see if your conditionals matched up.

**Comment by kiphale** [comment]
"Ratification or accession signifies an agreement to be legally bound by the terms of the Convention. Though accession has the same legal effect as ratification, the procedures differ. In the case of ratification, the State first signs and then ratifies the treaty. The procedure for accession has only one step—it is not preceded by an act of signature."

**Comment by budparr** [comment]
Rule #1 
If a country has State Party switch TRUE
AND
They have a "Date Signed"
the result is "**Ratified [date]**"

IF
If a country has State Party switch TRUE
AND
They DO NOT have a "Date Signed"
but DO have "Date Ratified or Acceded"
the result is "** Accession [date]**"

**Comment by budparr** [comment]
@kiphale please review when you return from lunch and let me know if it's right (if so closing the issue will suffice).

**Comment by kiphale** [comment]
Yes...perfect!
